src/transformers/adapters/model_mixin.py

```diff
@@ -188,7 +193,7 @@ def load_embeddings(self, path: str, name: str):
         self.set_active_embeddings(name)
         return tokenizer
 
-    def add_embeddings(self, name, tokenizer, reference_embedding=None, reference_tokenizer=None, embedding_dim=None):
```

Comment:
I'm not sure if the `embedding_dim` parameter can be removed without issues (as it was available before this PR). If we want to remove it, also remove the method docs of this param below.

Reply:
You're right, I added the parameter again